x-pack/plugins/siem/server/lib/detection_engine/rule_actions/saved_object_mappings.ts

```diff
@@ -4,37 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { SavedObjectsType } from '../../../../../../../src/core/server';
+
 export const ruleActionsSavedObjectType = 'siem-detection-engine-rule-actions';
 
 export const ruleActionsSavedObjectMappings = {
-  [ruleActionsSavedObjectType]: {
-    properties: {
-      alertThrottle: {
-        type: 'keyword',
-      },
-      ruleAlertId: {
-        type: 'keyword',
-      },
-      ruleThrottle: {
-        type: 'keyword',
-      },
-      actions: {
-        properties: {
-          group: {
-            type: 'keyword',
-          },
-          id: {
-            type: 'keyword',
-          },
-          action_type_id: {
-            type: 'keyword',
-          },
-          params: {
-            dynamic: true,
-            properties: {},
-          },
+  properties: {
+    alertThrottle: {
+      type: 'keyword',
+    },
+    ruleAlertId: {
+      type: 'keyword',
+    },
+    ruleThrottle: {
+      type: 'keyword',
+    },
+    actions: {
+      properties: {
+        group: {
+          type: 'keyword',
+        },
+        id: {
+          type: 'keyword',
+        },
+        action_type_id: {
+          type: 'keyword',
+        },
+        params: {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          dynamic: true as any,
```

Comment:
@patrykkopycinski FYI this isn't officially supported by New Platform; you can see [here](https://github.com/elastic/kibana/blob/master/src/core/server/ui_settings/saved_objects/ui_settings.ts#L27-L30) that it's a special case for UI Settings. If this is/was a temporary thing, perhaps we can make it a little more strict, nw?